spec.ts

```diff
@@ -0,0 +1,83 @@
+import { browser, element, by } from 'protractor';
+
+import { YM } from "./epochconverter/convert";
+import { SEC } from './epochconverter/convert_sdhm';
+import { DTG } from './epochconverter/timestamp';
+import { YMDHMS } from './epochconverter/ymdhms_timestamp';
+import * as moment from 'moment';
+
+
+describe('Epochconverter with protractor -', function() {
+
+	// ########## Mein ToDo ##########
+	// GMT zu Local time wechseln
+	// Timestamp to Human date fehlt noch
+	// Jeweils Rückkgabe nach Buttonklick auslesen
+	// ###############################
+
+	// Date and time
+	let year = moment().format('Y');
+	let month = moment().format('M');
+	let day = moment().format('D');
+	let hours_utc = moment().utc().format('H');
+	let hours = moment().format('H');
+	let minutes = moment().format('m');
+	let seconds = moment().format('s');
+
+	// letiables for test 
+	let ymdhms_timestamp_year = element(by.id("hf-d1")).element(by.className("dateform year"));
+	let ymdhms_timestamp_month = element(by.id("hf-d2")).element(by.className("dateform month"));
+	let ymdhms_timestamp_day = element(by.id("hf-d3")).element(by.className("dateform day"));
+	let ymdhms_timestamp_hours = element(by.name("hh"));
+	let ymdhms_timestamp_minutes = element(by.name("mn"));
+	let ymdhms_timestamp_seconds = element(by.name("ss"));
+	let timestamp = element(by.id("fs")).element(by.id("rcinput"));
+	let convert_year = element(by.id("br-d1")).element(by.className("dateform year"));
+	let convert_month = element(by.id("br-d2")).element(by.className("dateform month"));
+	let convert_sdhm = element(by.id("tc")).element(by.id("scinput"));
+
+	// Test case Convert epoch to human readable date and vice versa
+    beforeEach(function () {
+        browser.waitForAngularEnabled(false);
+        browser.get('https://www.epochconverter.com/');
+    });
+
+	// DONE!
+	it('TC 1.1. Check date & time', () => {
```

Comment:
Testfallname unklar -> Bessere Benennung, was wirklich getestet wird

Reply:
in Check current date & time umgenannt, es soll aktuelle Datum und Uhrzeit testen.